Cogs.Publishers/OwlPublisher.cs

```diff
@@ -55,6 +55,7 @@ public void Publish(CogsModel model)
             res.AppendLine(@"   <owl:Ontology rdf:about=""http://www.semanticweb.org/clement/ontologies/2017/6/" + projName + @"""/>");

```

Comment:
we should allow a target namespace, or a default namespace, instead of http://www.semanticweb.org/clement/ontologies/2017/6/

See the xsd publisher for an example of a command line input for a target namespace in the Program.cs